src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costRentPerDay = 40;
+  const basicDiscount = 20;
+  const extraDiscount = 50;
+
+  if (days > 6) {
+    return days * costRentPerDay - extraDiscount;
+  }
+
+  if (days > 2 && days < 7) {
+    return days * costRentPerDay - basicDiscount;
+  }
+
+  return days * costRentPerDay;
```

Comment:
looks like a new var

Reply:
i thought it`s already too much vars for a such small task)
added